appveyor.yml

```diff
@@ -44,40 +27,45 @@ environment:
     - platform: win32
       ADDR_MODEL: 32
       ARCH: win32-msvc14
-      MSVCVERSION: 14.0
+      MSVCVERSION: 14.2
       PYTHONPATH: c:\Python37-x64\
       PY_VER: 37
       BOOST_CFG: >-
           using python : 3.7 : c:/python37/python.exe : c:/python37/include : c:/python37/libs ;
     - platform: x64
       ADDR_MODEL: 64
       ARCH: x64-msvc14
-      MSVCVERSION: 14.0
+      MSVCVERSION: 14.2
       PYTHONPATH: c:\Python37-x64\
       PY_VER: 37
       BOOST_CFG: >-
           using python : 3.7 : c:/python37-x64/python.exe : c:/python37-x64/include : c:/python37-x64/libs ;
+          
+    - platform: win32
+      ADDR_MODEL: 32
+      ARCH: win32-msvc14
+      MSVCVERSION: 14.2
+      PYTHONPATH: c:\Python37-x64\
```

Comment:
Should this be `c:\Python38\`?

Reply:
BTW I think there is a mistake in the current version line 48: `PYTHONPATH: c:\Python37-x64\` should be `PYTHONPATH: c:\Python37\`